examples/CERA_workflow.ipynb

```diff
@@ -40,8 +40,9 @@
    },
    "outputs": [],
    "source": [
-    "world = gpd.read_file(gpd.datasets.get_path('naturalearth_lowres'))\n",
-    "us = world[world.name.isin(['United States of America', 'Puerto Rico'])]\n",
+    "world = gpd.read_file('https://s3.us-west-2.amazonaws.com/naciscdn.org/naturalearth/110m/cultural/110m_cultural.zip')\n",
+    "\n",
+    "us = world[world.NAME_EN.isin(['United States of America', 'Puerto Rico'])]\n",
```

Comment:
@tomsail this is due to update in geopandas where they removed the datasets module in their new release.